lib/env.h

```diff
@@ -43,10 +43,14 @@
 #endif
 
 // Detect word size:
-#ifdef _INTEGRAL_MAX_BITS
+#if defined(_INTEGRAL_MAX_BITS)
 #  define BASE64_WORDSIZE _INTEGRAL_MAX_BITS
-#else
+#elif defined(__WORDSIZE)
 #  define BASE64_WORDSIZE __WORDSIZE
+#elif defined (__LONG_WIDTH__)
+#  define BASE64_WORDSIZE __LONG_WIDTH__
+#else
+#  error BASE64_WORDSIZE_NOT_DEFINED
```

Comment:
This will print the string `BASE64_WORDSIZE_NOT_DEFINED`. Is that intentional or did you intend to define a macro of that name?